README.rst

```diff
@@ -93,8 +95,8 @@ Markers
 
     map_1 = folium.Map(location=[45.372, -121.6972], zoom_start=12,
                        tiles='Stamen Terrain')
-    map_1.simple_marker([45.3288, -121.6625], popup='Mt. Hood Meadows')
-    map_1.simple_marker([45.3311, -121.7113], popup='Timberline Lodge')
+    folium.Marker([45.3288, -121.6625], popup='Mt. Hood Meadows').add_to(map_1)
+    folium.Marker([45.3311, -121.7113], popup='Timberline Lodge').add_to(map_1)
```

Comment:
@BibMartin I have @qingkaikong here at scipy sprinting on folium. Here is his first PR (of many :wink:).